netcfgbu/cli/root.py

```diff
@@ -32,15 +43,30 @@ def invoke(self, ctx):
 
 
 class WithInventoryCommand(click.Command):
+    """Custom Click command that preloads configuration and inventory.
+
+    This function loads the necessary configuration & inventory before invoking the specified
+    command. It also initializes SSH debugging & jumphost setup if these options are configured.
+    """
+
     def invoke(self, ctx):
+        """Invokes the command after loading the configuration and inventory.
 
+        Args:
+            ctx: Click context object.
+
+        Raises:
+            Exception: If there is an error loading the configuration, inventory,
+            or initializing jumphosts.
+        """
         try:
             app_cfg = ctx.obj["app_cfg"] = _config.load(fileio=ctx.params["config"])
 
             if debug_ssh_lvl := ctx.params.get("debug_ssh"):  # pragma: no cover
-                from asyncssh import logging as assh_lgr
                 import logging
 
+                from asyncssh import logging as assh_lgr
+
                 assh_lgr.set_log_level(logging.DEBUG)
```

Comment:
**Suggestion:** Refactor the `WithInventoryCommand.invoke` method to separate concerns, specifically extracting the SSH debugging setup into a separate method. This will make the code cleaner and easier to maintain. [maintainability, importance: 7]
```suggestion
self.setup_ssh_debugging(ctx.params.get("debug_ssh"))
```